app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription/reg06_eori_already_linked.scala.html

```diff
@@ -17,20 +17,20 @@
 @import org.joda.time.DateTime
 @import uk.gov.hmrc.eoricommoncomponent.frontend.forms.FormUtils.dateTimeFormat
 @import uk.gov.hmrc.eoricommoncomponent.frontend.views.html._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName._
 
 
 @this(layout_di: layout)
 @(name: String, processedDate: DateTime)(implicit messages: Messages, request: Request[_])
-@layout_di(messages("cds.reg06.outcome.eori-already-linked.title-and-heading")) {
+@layout_di(messages("cds.reg06.outcome.eori-already-linked.title-and-heading", shortName, name)) {
```

Comment:
Are you sure here should be `shortName` and `name`?

cds.reg06.outcome.eori-already-linked.title-and-heading=The {0} application for {1} has been unsuccessful

What I understand based on this that this message will be The ATaR application for Adavance Tariff Rulings has been unsuccessful